nethserver-suricata.spec

```diff
@@ -54,6 +54,7 @@ rm -rf $RPM_BUILD_ROOT
 %defattr(-,root,root)
 %dir %{_nseventsdir}/%{name}-update
 /usr/libexec/nethserver/api/%{name}/
+%attr(0440,root,root) /etc/sudoers.d/20_nethserver_suricata
```

Comment:
Did you check the resulting RPM?
I see a warning and I'm afraid the `%attr` macro is ignored here: https://travis-ci.com/NethServer/nethserver-suricata/builds/148062234#L324

To set the file attributes at the first file occurrence see line 42.

Reply:
I tested it locally, by creating a rpm with nethserver-mock and it seems working fine.
Now I tested nethserver-openvpn PR and after installing the rpm `visudo -c` returns OK.